src/test/java/org/opentripplanner/middleware/utils/ItineraryUtilsTest.java

```diff
@@ -43,32 +49,66 @@
+        2020, 8, 13, 2, 59, 59)); // Aug 13, 2020 2:59:59 AM, considered to be Aug 12.
+    public static final long _2020_08_13__03_00_00 = otpDateTimeAsEpochMillis(LocalDateTime.of(
+        2020, 8, 13, 3, 0, 0)); // Aug 13, 2020 3:00:00 AM
+    public static final long _2020_08_13__23_59_59 = otpDateTimeAsEpochMillis(LocalDateTime.of(
+        2020, 8, 13, 23, 59, 59)); // Aug 13, 2020 11:59:59 PM
+    public static final long _2020_08_14__02_59_59 = otpDateTimeAsEpochMillis(LocalDateTime.of(
+        2020, 8, 14, 2, 59, 59)); // Aug 14, 2020 2:59:59 AM, considered to be Aug 13.
+    public static final long _2020_08_14__03_00_00 = otpDateTimeAsEpochMillis(LocalDateTime.of(
+        2020, 8, 14, 3, 0, 0)); // Aug 14, 2020 3:00:00 AM
+
+    /** Contains an OTP response with an itinerary. */
+    private static final OtpDispatcherResponse OTP_DISPATCHER_PLAN_RESPONSE =
+        initializeMockPlanResponse(TEST_RESOURCE_PATH + "persistence/planResponse.json");
+    /** Contains an OTP response with no itinerary found. */
+    private static final OtpDispatcherResponse OTP_DISPATCHER_PLAN_ERROR_RESPONSE =
+        initializeMockPlanResponse(TEST_RESOURCE_PATH + "persistence/planErrorResponse.json");
+    /** Contains the verified itinerary set for a trip upon persisting. */
+    private static final Itinerary DEFAULT_ITINERARY = OTP_DISPATCHER_PLAN_RESPONSE.getResponse().plan.itineraries.get(0);
+    public static final ZoneId OTP_ZONE_ID = DateTimeUtils.getOtpZoneId();
 
     @BeforeAll
-    public static void setup() throws IOException {
+    public static void setup() {
         TestUtils.mockOtpServer();
+    }
 
+    public static OtpDispatcherResponse initializeMockPlanResponse(String path) {
         // Contains an OTP response with an itinerary found.
         // (We are reusing an existing response. The exact contents of the response does not matter
         // for the purposes of this class.)
-        String mockPlanResponse = FileUtils.getFileContents(
-            TEST_RESOURCE_PATH + "persistence/planResponse.json"
-        );
-        // Contains an OTP response with no itinerary found.
-        String mockErrorResponse = FileUtils.getFileContents(
-            TEST_RESOURCE_PATH + "persistence/planErrorResponse.json"
-        );
+        String mockPlanResponse = null;
+        try {
+            mockPlanResponse = FileUtils.getFileContents(path);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        return new OtpDispatcherResponse(mockPlanResponse, DEFAULT_PLAN_URI);
+    }
 
-        otpDispatcherPlanResponse = new OtpDispatcherResponse(mockPlanResponse, DEFAULT_PLAN_URI);
-        otpDispatcherPlanErrorResponse = new OtpDispatcherResponse(mockErrorResponse, DEFAULT_PLAN_URI);
-        defaultItinerary = otpDispatcherPlanResponse.getResponse().plan.itineraries.get(0);
+    public static List<OtpResponse> createMockOtpResponsesForTripExistence() {
```

Comment:
This method is not used anywhere in this file. Instead, it is referenced from `ApiUserFlowTest` and `GetMonitoredTripsTest`. To me, this seems to be some kind of function that ought to be placed in some kind of Util class so that there are less util functions in classes like these that contain tests. This should probably be moved to some class with mainly Util functions. I went ahead and made https://github.com/ibi-group/otp-middleware/pull/119 that reogranizes a bunch of test files. Assuming that PR is accepted, the best spot for this would be within the `OtpTestUtils` class. However, in order to move this along, I think it'd be fine to just chuck this one into the `TestUtils` class for now and I'd be fine with reorganizing as a part of #119.

Reply:
Updated in https://github.com/ibi-group/otp-middleware/pull/102/commits/98f91c7d7043e0039a39a1855dc3e40befbf178e. I moved it to `TestUtils`.